noir-projects/noir-contracts/contracts/test/returning_tuple_contract/src/main.nr

```diff
@@ -0,0 +1,47 @@
+use aztec::macros::aztec;
+
+// Tests that Aztec.nr handles returning tuples correctly.
+#[aztec]
+pub contract ReturningTuple {
```

Comment:
I really like the thoroughness of creating a test contract with different variants, but it'd be good to test that this is also handled correctly by the rest of the stack, e.g. txe and e2e tests that call these fns. Not sure where that'd live, do we have tests for handling return values elsewhere?